types.go

```diff
@@ -374,6 +374,23 @@ func (m *MerkleTreeLeaf) Precertificate() (*x509.Certificate, error) {
 	return x509.ParseTBSCertificate(m.TimestampedEntry.PrecertEntry.TBSCertificate)
 }
 
+// APIEndpoint is a string that represents one of the Certificate Transparency
+// Log API endpoints.
+type APIEndpoint string
+
+// Certificate Transparency Log API endpoints - matches the URI paths without
+// the "/ct/v1/" prefix.
```

Comment:
Maybe add a warning to keep these in sync with `Path` strings, maybe even another warning above the `Path` block, so whichever block is modified the warning is noticeable?

Reply:
Done.